.changeset/fuzzy-dolls-shake.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-catalog-import': patch
```

Comment:
Good that you pointed out that this is breaking in catalog-import. In this case this should be minor 😅 

```suggestion
'@backstage/plugin-catalog-import': minor
```